### ISSUE TITLE
reproduce performance

### ISSUE THREAD

**Issue by purse1996** [lxtGH/PFSegNets#5]
Follow the experiment setup of the paper,  I crop the image into a  ﬁxed size of  (896,  896)  using a  sliding window striding  512  pixels. And use sh scripts/pointflow/train_iSAID_pfnet_r50.sh to train my model. I train the model using 8 RTX-3090. 
The result is best record: [val loss 0.04381], [acc 0.99043], [acc_cls 0.76056], [mean_iu 0.66368], [fwavacc 0.98220], [epoch 15],  where mIoU is below 0.5%.  
Can you give some suggestions? Thank you.

**Comment by lxtGH** [comment]
0.5\% is a ramdom noise. You can try to re-train it or performe the longer training such as 32 epoch training.
